package/gluon-core/luasrc/lib/gluon/upgrade/200-wireless

```diff
@@ -225,6 +226,13 @@ util.foreach_radio(uci, function(radio, index, config)
 		end
 	end
 
+
+	if beacon_interval then
+		uci:set('wireless', radio_name, 'beacon_int', beacon_interval)
+	else
+		uci:delete('wireless', radio_name, 'beacon_int')
+	end
```

Comment:
The if/else can be removed - unlike the old *luci.model.uci*, *simple-uci*'s `uci:set()` will delete the option when `nil` is passed.

Reply:
This is good to know. I assume I've did this for a long time on multiple occasions.

Removed the redundant block.